backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -18,17 +18,27 @@
 
 package net.skyscanner.backpack.compose.theme
 
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Colors
 import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Shapes
 import androidx.compose.material.Typography
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.CompositionLocalProvider
 import androidx.compose.runtime.staticCompositionLocalOf
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
+import net.skyscanner.backpack.compose.tokens.BpkBorderRadius
+import net.skyscanner.backpack.compose.tokens.BpkColor
+import net.skyscanner.backpack.compose.tokens.BpkColors
 import net.skyscanner.backpack.compose.tokens.BpkTypography
 
-val LocalBpkTypography = staticCompositionLocalOf {
-  BpkTypography()
+private val LocalBpkTypography = staticCompositionLocalOf<BpkTypography> {
+  error("Wrap you content with BpkTheme {} to get access to LocalBpkTypography")
+}
+private val LocalBpkColors = staticCompositionLocalOf<BpkColors> {
```

Comment:
Private is needed we as only want to allow customisation via BpkTheme – it's should be closed for any other non-api stated modifications.